### PR TITLE
Final yapf setup

### PR DESCRIPTION
**Pull request by jackdavidweber**
[yapf](https://opensource.google/projects/yapf) is a Google open source formatter for python files. In this PR, I did the following:

- Used yapf to automatically apply the google style guide to our backend repository while excluding directories that do not need to be formatted (i.e. experimental).
- Edited yaml file to automatically check if our code meets google style guides on every push/PR.

In order to run yapf on your branch:
```
pip install yapf
yapf -ir --verbose --style "google" .
```

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -33,11 +35,13 @@ def post(self):
         response_input_lang = input_lang
 
         # Gets non-beta (fully supported) languages for automatic detection
-        fully_supported_lang_codes = ConverterRegistry.get_fully_supported_language_codes()
+        fully_supported_lang_codes = ConverterRegistry.get_fully_supported_language_codes(
+        )
```
Comment (cjoshea9) [inline app.py:39]: This seems to be wrong

File: java/code_to_gast/java_assign.py [hunk 2]
```diff
@@ -1,20 +1,24 @@
...
     # var name stored as string but we don't want to return gast string node
     gast["varId"] = {"type": "name", "value": node.name}
     return gast
 
+
 """
 Handles increment and decrement operators
 """
+
+
```
Comment (cjoshea9) [inline java/code_to_gast/java_assign.py:19]: I don't think we want space between our doc string and function

File: app.py (hunk 1 above)
Comment (Danish21) [reply app.py:39]: +1. This seems strange.

File: java/code_to_gast/java_assign.py (hunk 2 above)
Comment (Danish21) [reply java/code_to_gast/java_assign.py:19]: +1. Is there a way to configure this?

Comment (g-harel) [reply java/code_to_gast/java_assign.py:19]: Python doc strings usually go _inside_ the function, I think that might be why it's getting confused.

File: app.py (hunk 1 above)
Comment (g-harel) [reply app.py:39]: Agreed, seems like a quirk of setting a fixed line cutoff. Not sure there's much to do about it, maybe could move the `.get_fully_...` to the next line.

**Pull request merged**